### ISSUE TITLE
Request: navigating to home screen refreshes feeds

### ISSUE THREAD

**Issue by cyranix** [samuelclay/NewsBlur#1346]
I use NewsBlur across two computers, switching between them somewhat often (perhaps abnormally so). I also access NewsBlur on my (Android) phone. As a result, I encounter stale state fairly often, and end up doing a full page reload to resync, since I operate almost exclusively in Unread mode. 

If it wouldn't be too impactful on the backend, I would love for the behavior of the home screen icon to include "refresh the state for feeds (if the state hasn't been updated in the last _n_ minutes)".

**Comment by nriley** [comment]
For what it's worth, you don't need to do a full reload to resync. Shift+E (Open Everything) will refresh without reloading — there may be other ways to do it, but it works for me.

**Comment by samuelclay** [comment]
Everything should be in sync when you come back to the computer. As in unread counts. Any open feeds won't show new stories as they come in because that's too distracting, but the feed title will flash when new stories come in.

If you're finding unread counts are stale, can you let me know if you see the real-time indicator? It's a little icon on the web dashboard at the top of the main column, below your name and next to "Premium Account". That should tell you if you're connected to real-time notifications. If not, then NewsBlur will be sync'ing unread counts every 60 seconds.

**Comment by cyranix** [comment]
@samuelclay, I do see the real-time indicator (the subtle back-and-forth-y one). 

My steps to repro:

* NewsBlur is open on computers A and B, both using Unread mode in the sidebar.
* I read an article on computer A, dropping the number of unread items from 1 to 0.
* Without refreshing the page, computer B does not mirror this update and continues to show 1 unread story in that feed.
  * Neither `Shift-E` nor `R` cause the correct state to be shown.
* Upon clicking into the relevant feed, no items are shown in the feed, and yet the unread count still shows as 1.
  * Pressing `R` does create a UI effect indicating that the refresh is happening, but the sidebar still shows 1 unread story.

**Comment by cyranix** [comment]
This could theoretically be a regression of an issue I filed years ago (#62), or could just be the same symptom with a different cause.

**Comment by samuelclay** [comment]
Really what this needs is a better return-to-browser notification. I'm assuming the computer is a laptop that is closed? So it's stale because it's just waking up and has not refreshed yet. 

Somewhere I've seen a JS notification library that I could use...

**Comment by cyranix** [comment]
No, this happens with two laptops open side-by-side (work laptop and personal).

If it's helpful, I wouldn't mind scheduling a screenshare session to help with understanding the reproducibility. Just depends on where this falls on your priority list.

**Comment by samuelclay** [comment]
Can you open the dev console and see what it says when you read on the other device? It should provide valuable debugging info about seeing stories being read elsewhere.

**Comment by samuelclay** [comment]
Closing — the iOS app has an auto-refresh timer mechanism and pull-to-refresh on the feed list. If feeds are still not refreshing as expected, please open a new issue with specifics.
